### PR TITLE
Create CH-BS

### PR DESCRIPTION
**Pull request by AudreyIATI**
Related to [Issue 501](https://github.com/org-id/register/issues/501)

### CODE REVIEW

File: lists/ch/ch-bs-cr.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "CH"
+  ],
+  "subnationalCoverage": [
+    "CH-BS"
+  ],
+  "structure": [
+    "company"
+  ],
+  "sector": [],
+  "code": "CH-BS",
```
Comment (kd-ods) [inline lists/ch/ch-bs-cr.json:20]: Please change this to CH-BS-CR. (I've already edited the filename and directory to match this.)

**Review by kd-ods — CHANGES_REQUESTED** [review]
There are a few things to amend here.

**Comment by AudreyIATI** [comment]
Changes made - let me know if I missed anything.

Thanks,
Audrey

**Review by kd-ods — APPROVED** [review]

**Comment by andylolz** [comment]
It looks like this is the only code to contain more than one hyphen. I wonder if the second hyphen could be replaced by an underscore? I.e. `CH-BS_CR`. It looks like that’s the approach taken with other codes (e.g. `AE-RK_CR`; `BE-BCE_KBO`; `CA-CRA_ACR` etc).

I checked [the guidance](http://docs.org-id.guide/en/latest/metadata/#list-name-code), and it doesn’t look like there’s anything to say a hyphen shouldn’t be used in either part of the code. But hyphens are used to separate the two parts of the register code, as well as to separate the two parts of the organisation ID that these codes are used in. Adding additional hyphens makes it ambiguous as to which part of the code is which.

**Comment by kd-ods** [comment]
Thanks @andylolz - I'll take a look at this. I get the point about ambiguous separability.

**Comment by kd-ods** [comment]
@andylolz - I'm proposing this [edit to the org-id handbook](https://github.com/org-id/handbook/pull/25) which would mean changing this particular org list code to CH_BS-CP. Does that look OK to you? Do comment on the handbook PR if that's more fitting.

**Comment by kd-ods** [comment]
@andylolz - I'm proposing this [edit to the org-id handbook](https://github.com/org-id/handbook/pull/25) which would mean changing this particular org list code to CH_BS-CP. Does that look OK to you? Do comment on the handbook PR if that's more fitting.

**Comment by andylolz** [comment]
> which would mean changing this particular org list code to CH_BS-CP. Does that look OK to you?

@kd-ods Aha! Good catch – my underscore was misplaced! This is great, thanks. Fully support this proposal.

Thanks also for creating and linking to the proposed handbook edit – the explanation in your PR is really clear.

**Pull request merged**